src/main.cc

```diff
@@ -184,9 +184,16 @@ main(const int argc, char* argv[]) {
         .is_err();
   } catch (const structopt::exception& e) {
     if (argc > 1) {
+      int subcommand_index = 1;
```

Comment:
```suggestion
      usize subcommand_index = 1;
```

`usize` is preferred for index values.

Reply:
I agree with you, but `argc` is `int` . Would I cast `argc` as `usize` too?

Reply:
Did the comparison cause an implicit conversion error?

Reply:
[It causes just a warning](https://wandbox.org/permlink/px9DKrP6ZpP7D1Ny) (and it must be safe), but IMO **warning SHOULD be suppressed in appropriate methods** . This is just a my opinion, so if you don't think so, I will pick above suggestion.

Reply:
Alright, thank you. Let's use `i32` instead of `int`.

```suggestion
      i32 subcommand_index = 1;
```